base/metrics/metrics.go

```diff
@@ -15,10 +15,23 @@ var (
 	}, []string{"type"})
 )
 
+var (
+	EngineVersion = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Help:      "Patchman project deployment information",
+		Namespace: "patchman_engine",
+		Subsystem: "core",
+		Name:      "info",
+	}, []string{"version"})
+)
+
+var ENGINEVERSION = "v1.4.1"
+
 func init() {
 	if os.Getenv("KAFKA_ADDRESS") != "" {
 		prometheus.MustRegister(KafkaConnectionErrorCnt)
 	}
+	prometheus.MustRegister(EngineVersion)
+	EngineVersion.With(prometheus.Labels{"version":ENGINEVERSION}).Set(1)
```

Comment:
Good, but `EngineVersion.WithLabelValues(ENGINEVERSION).Set(1)` is easier.

Reply:
fixed